src/api/workflow-api.js

```diff
@@ -1,7 +1,9 @@
 import { mainHttpClient } from '../httpClient'
 
 export default {
-  getAllWorkflows: () => {
-    return mainHttpClient.get('/workflows?page_size=100')
+  getAllWorkflows: pageToken => {
+    return mainHttpClient.get(
+      `/workflows?page_size=100${pageToken ? `&page_token=${pageToken}` : ''}`
```

Comment:
Instead of manipulating the string sent as the `url` param of `axios.get` method, you could pass a `config` object as the second param, and there you have a `params` property that axios itself converts to the appropriate string.
You can try something like this:
```js
const params = {
  page_size: 100
}
if (pageToken) {
  params.page_token = pageToken
}
mainHttpClient.get('/workflows', { params })
```